.github/workflows/vmtest.yml

```
@@ -0,0 +1,25 @@
+name: vmtest
+on:
+  workflow_run:
+    workflows: [Build]
```

```suggestion
    workflows: ["Build"]
```

This could be the culprit. We need to try.